src/flightlog_fields_presenter.js

```diff
@@ -1016,10 +1016,11 @@ const DEBUG_FRIENDLY_FIELD_NAMES_INITIAL = {
   },
   ATTITUDE: {
     "debug[all]": "Attitude",
-    "debug[0]": "IMU Gain",
-    "debug[1]": "EZ_EF",
-    "debug[2]": "GroundSpeedError",
-    "debug[3]": "VelocityFactor",
+    "debug[0]": "Roll angle",
+    "debug[1]": "Pitch angle",
+    "debug[2]": "Ground speed factor",
+    "debug[3]": "Heading error",
+    "debug[7]": "Error gain",
```

Comment:
This will break backwards compatibility.

Afaik it was changed in firmware 4.5.0 in https://github.com/betaflight/betaflight/pull/12900

Reply:
I've understanded. Thank's